.github/workflows/auto-release-branch-e2e.yaml

```diff
@@ -0,0 +1,108 @@
+name: Auto Build e2e-runner for New Release Branch
+
+on:
+  create:
+  push:
+    branches:
+      - 'release-1.**'
```

Comment:
do you want this to trigger on ONLY pushes to the branch for the ibm dockerfile? Or on all files in the branch?

```suggestion
      - 'release-1.**'
   paths:
      - '.ibm/images/Dockerfile'
```

Reply:
The purpose of this workflow is to build the E2E runner only for release-1.* branches, and only when the branch is initially created.
Subsequent commits to the branch should not trigger a rebuild of the E2E runner.

We already have a separate workflow that builds the E2E runner whenever changes are made to the .ibm/Dockerfile.